src/libraries/System.Private.CoreLib/src/System/Type.cs

```diff
@@ -132,6 +132,27 @@ public virtual Type[] GetGenericParameterConstraints()
         [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
         public ConstructorInfo? GetConstructor(Type[] types) => GetConstructor(BindingFlags.Public | BindingFlags.Instance, null, types, null);
 
+        /// <summary>
+        /// Searches for a constructor whose parameters match the specified argument types, using the specified binding constraints.
+        /// </summary>
+        /// <param name="bindingAttr">
+        /// A bitwise combination of the enumeration values that specify how the search is conducted.
+        /// -or-
+        /// Default to return null.
+        /// </param>
+        /// <param name="types">
+        /// An array of Type objects representing the number, order, and type of the parameters for the constructor to get.
+        /// -or-
+        /// An empty array of the type <see cref="Type"/> (that is, Type[] types = Array.Empty{Type}()) to get a constructor that takes no parameters.
+        /// -or-
+        /// <see cref="EmptyTypes"/>.
+        /// </param>
+        /// <returns>
+        /// A <see cref="ConstructorInfo"/> object representing the constructor that matches the specified requirements, if found; otherwise, null.
+        /// </returns>
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.NonPublicConstructors)]
+        public ConstructorInfo? GetConstructor(BindingFlags bindingAttr, Type[] types) => GetConstructor(bindingAttr, null, types, null);
```

Comment:
Normally `null` should specify a parameter name for readability, but I guess it's OK to leave as-is since you're just using the existing pattern in this file.